sqlmesh/core/loader.py

```diff
@@ -38,6 +39,7 @@
     from sqlmesh.core.context import Context
 
 
+# maybe move these two methods to context
```

Comment:
is this a fixme a todo or shouldn't be here?

Reply:
a todo

Reply:
so let's probably name it as such